pymc_marketing/mmm/evaluation.py

```diff
@@ -0,0 +1,221 @@
+#   Copyright 2024 The PyMC Labs Developers
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+import arviz as az
+import matplotlib.pyplot as plt
+import numpy as np
+import pymc as pm
+from loguru import logger
+from sklearn.metrics import (
+    mean_absolute_error,
+    mean_absolute_percentage_error,
+    root_mean_squared_error,
+)
+
+from pymc_marketing.mmm.delayed_saturated_mmm import MMM
+
+
+# Same error metric as Robyn
+def nrmse(y_true, y_pred):
+    """
+    Normalized Root Mean Square Error. Normalization allows for comparison across
+    different data sets and methodologies. e.g. NRMSE is one of the key metrics used
+    in Robyn MMMs.
+    Args:
+        y_true ([np.array]): test samples
+        y_pred ([np.array]): predicted samples
+    Returns:
+        [float]: normalized root mean square error
```

Comment:
Can you please change the docstring format, see https://numpydoc.readthedocs.io/en/latest/format.html#parameters :)

Reply:
See for example https://github.com/pymc-labs/pymc-marketing/blob/39f23367efe6fc512f3d0558cb97712d68d144f1/pymc_marketing/mmm/delayed_saturated_mmm.py#L227-L256

Reply:
Ok fixed this part at least I think ☺️